src/platform-includes/crons/setup/java.spring-boot.mdx

```diff
@@ -0,0 +1,65 @@
+If you are using [Quartz](http://www.quartz-scheduler.org/), please see <PlatformLink to="/configuration/integrations/quartz/">our Quartz integration</PlatformLink>. You may also [send check-ins manually](/platforms/java/crons/).
+
+## Check-Ins (Recommended)
+
+Check-in monitoring allows you to track a job's progress by completing two check-ins: one at the start of your job and another at the end of your job. This two-step process allows Sentry to notify you if your job didn't start when expected (missed) or if it exceeded its maximum runtime (failed).
```

Comment:
Maybe add something like about the `@SentryCheckin` Annotation. Not sure if it is clear to the reader by it just being referred to in the code sample.